snakemake_executor_plugin_azure_batch/__init__.py

```diff
@@ -176,6 +173,14 @@ class ExecutorSettings(ExecutorSettingsBase):
             "env_var": False,
         },
     )
+    container_image: Optional[str] = field(
```

Comment:
container_image is a global snakemake option. The executor does not need its own: https://github.com/snakemake/snakemake/blob/aa1c2839165fc03d20e5a28657b2e1fa7ffdf80e/snakemake/settings.py#L349